scipy/stats/tests/test_multivariate.py

```diff
@@ -1635,6 +1638,187 @@ def test_haar(self):
+
+    def test_mvt_with_high_df_is_approx_normal(self):
+        # `normaltest` returns the chi-squared statistic and the associated
+        # p-value. The null hypothesis is that `x` came from a normal
+        # distribution, so a low p-value represents rejecting the null, i.e.
+        # that it is unlikely that `x` came a normal distribution.
+        P_VAL_MIN = 0.1
+
+        dist = multivariate_t(0, 1, df=100000, seed=1)
+        samples = dist.rvs(size=100000)
+        _, p = normaltest(samples)
+        assert (p > P_VAL_MIN)
+
+        dist = multivariate_t([-2, 3], [[10, -1], [-1, 10]], df=100000,
+                              seed=42)
+        samples = dist.rvs(size=100000)
+        _, p = normaltest(samples)
+        assert ((p > P_VAL_MIN).all())
+
+        dist = multivariate_t(0, 1, df=np.inf, seed=7)
+        samples = dist.rvs(size=100000)
+        _, p = normaltest(samples)
+        assert (p > P_VAL_MIN)
+
+    def test_default_arguments(self):
+        dist = multivariate_t()
+        assert_equal(dist.loc, [0])
+        assert_equal(dist.shape, [[1]])
+        assert (dist.df == 1)
+
+    def test_reproducibility(self):
+        rng = np.random.RandomState(4)
+        loc = rng.random(3)
+        tmp = rng.random((20, 3))
+        shape = np.matmul(tmp.T, tmp)
+        dist1 = multivariate_t(loc, shape, df=3, seed=2)
+        dist2 = multivariate_t(loc, shape, df=3, seed=2)
+        samples1 = dist1.rvs(size=10)
+        samples2 = dist2.rvs(size=10)
+        assert_equal(samples1, samples2)
+
+    def test_scalar_list_and_ndarray_arguments(self):
+        dist = multivariate_t(-1, 2, 3)
+        assert(dist.loc == np.array([-1]))
+        assert(dist.shape == np.array([2]))
+        assert(dist.df == 3)
+
+        dist = multivariate_t([-1], [2], 3)
+        assert(dist.loc == np.array([-1]))
+        assert(dist.shape == np.array([2]))
+        assert(dist.df == 3)
+
+        dist = multivariate_t(np.array([-1]), np.array([2]), 3)
+        assert(dist.loc == np.array([-1]))
+        assert(dist.shape == np.array([2]))
+        assert(dist.df == 3)
+
+    def test_allow_singular(self):
+        loc = np.zeros(4)
+        shape = np.eye(4)
+        df = 1
+
+        try:
+            multivariate_t(loc, shape, df, allow_singular=False)
+        except np.linalg.LinAlgError:
+            pytest.fail("Test failed by unexpectedly raising `LinAlgError`.")
+
+        # Make shape singular and verify error was raised.
+        shape[2, 2] = 0
+        allow_singular = False
+        seed = 0
+        assert_raises(np.linalg.LinAlgError,
+                      multivariate_t,
+                      loc, shape, df, allow_singular, seed)
+
+        # Check that singular `shape` is when when `allow_singular=True`.
+        try:
+            multivariate_t(loc, shape, df, allow_singular=True)
+        except np.linalg.LinAlgError:
+            pytest.fail("Test failed by unexpectedly raising `LinAlgError`.")
+
```

Comment:
similar as above.

Reply:
Please see [this comment](https://github.com/scipy/scipy/pull/12582/files/b16030a2e5c423231106c15ca5bc093e475a0c32#r493549941).